Examples/ArcGISToolkitExamples/JobManagerExample.swift

```diff
@@ -326,13 +350,17 @@ class JobManagerExample: TableViewController {
                 
                 // register the job with our JobManager shared instance
                 JobManager.shared.register(job: job)
+
+                // try to keep the app running in the background for this job if possible
+                let backgroundTaskIdentifier = self.startBackgroundTask()
                 
                 // start the job
                 job.start(
                     statusHandler: { [weak self] in
                         self?.jobStatusHandler(status: $0)
                     },
                     completion: { [weak self] in
+                        self?.endBackgroundTask(backgroundTaskIdentifier)
                         self?.jobCompletionHandler(result: $0, error: $1)
```

Comment:
I'm not sure how soon after a call to `endBackgroundTask` the system will suspend your app if it's in the background, you may want to call this as _late_ as possible when things are completed.

Reply:
good call

Reply:
fixed